lib/src/widgets/animations/ds_record_status.widget.dart

```diff
@@ -0,0 +1,64 @@
+import 'dart:async';
+
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+
+class DSRecordStatus extends StatefulWidget {
+  const DSRecordStatus({
+    super.key,
+  });
+
+  @override
+  State<DSRecordStatus> createState() => _DSRecordStatusState();
+}
+
+class _DSRecordStatusState extends State<DSRecordStatus> {
+  final shouldFadeIn = RxBool(true);
+  Timer? timer;
```

Comment:
a suggestion is to use AnimationController then its not necessary to use any timer or a RxBool

Reply:
Nice! I've done.